notebooks/scikit-learn/20_newsgroups_automl.ipynb

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
Although notebook focuses in specific task for AutoML do you think makes sense to add the same sections as other notebooks?

- Overview
- Dataset
- Objective 
- What's next?
https://github.com/GoogleCloudPlatform/cloudml-samples/blob/master/notebooks/tensorflow/getting-started-keras.ipynb